banzai_nres/settings.py

```diff
@@ -0,0 +1,34 @@
+from banzai_nres.images import NRESImage
+from banzai import settings
+from banzai.context import TelescopeCriterion
+import operator
+from banzai import bias, trim, dark, gain, bpm, qc
+import banzai_nres.bias as nres_bias
+import banzai_nres.dark as nres_dark
+from banzai_nres.flats import FlatStacker
+
+NRES_CRITERIA = [TelescopeCriterion('camera_type', operator.contains, 'NRES')]
+
+settings.ORDERED_STAGES = [bpm.BPMUpdater,
```

Comment:
Are you happy with this override? There could be a case for moving `ORDERED_STAGES` back into `main.py`. In particular, (in BANZAI imaging) I wanted to import `settings` in some module, but couldn't because the fact that `settings.py` imports every stage would result in a circular import.

Reply:
I agree with this. It is more transparent to me if the list of stages is defined in main.py where all the action starts.